st2common/setup.py

```diff
@@ -43,6 +43,7 @@
     packages=find_packages(exclude=['setuptools', 'tests']),
     scripts=[
         'bin/st2-bootstrap-rmq',
+        'bin/st2-cleanup-db',
```

Comment:
Is it a good idea to include this as part of standard distribution (need to also check st2-packages).

This is mostly meant for dev purposes and if we ship it with the actual package distribution I'm worried someone could accidentally run it and remove all data in the database.

So in short, I think we should make it more fail-safe - perhaps the best way is move it under tools/ or similar and not include it with packages.

Reply:
`st2ctl clean` relies on that file, therefore it should be part of the packaging.

There is a https://github.com/StackStorm/st2-packages/pull/494/ which adds symlink `/usr/bin/st2-cleanup-db` via `st2-packages`. I don't think we should do that, but "hide it" via `/opt/stackstorm/st2/bin`

Reply:
@armab i've closed the packaging PR StackStorm/st2-packages#494 and also updated `st2common/bin/st2ctl` to execute the command using the full path instead if relying on it being available in `$PATH`